### PR TITLE
Update backlog-refinement.md

### PR DESCRIPTION
**Pull request by TheETupper**
Removed broken link to scrumalliance.org I was unable to find an alternative link.

### CODE REVIEW

File: docs/050-how-we-work/agile-practices/backlog-refinement.md [hunk 1]
```diff
@@ -27,4 +27,3 @@ Backlog Refinement sessions are an important part of preparation for [Sprint Pla
 ## Additional Resources:
 
 - [Scrum Training Series Video](http://scrumtrainingseries.com/BacklogRefinementMeeting/BacklogRefinementMeeting.htm)
-- [Scrum Alliance Blog Post on Backlog Refinement](https://www.scrumalliance.org/community/articles/2014/october/product-backlog-refinement)
```
Comment (dmundra) [inline docs/050-how-we-work/agile-practices/backlog-refinement.md:30]: @TheETupper maybe this one https://resources.scrumalliance.org/Article/keys-effective-product-backlog-refinement could replace it?

Comment (TheETupper) [reply docs/050-how-we-work/agile-practices/backlog-refinement.md:30]: @dmundra I looked at that one too, and as I was reading I was getting conflicting info from other information in the handbook. A 3-hour meeting vs the 60-90 minute recommendation. I think that could be confusing.

Comment (dmundra) [reply docs/050-how-we-work/agile-practices/backlog-refinement.md:30]: Good points!

**Review by dmundra — APPROVED** [review]

File: docs/050-how-we-work/agile-practices/backlog-refinement.md (hunk 1 above)
Comment (dmundra) [reply docs/050-how-we-work/agile-practices/backlog-refinement.md:30]: Approved and please feel free to merge it!

**Pull request merged**